client/src/views/account/CreateAccountPage.vue

```diff
@@ -316,7 +316,7 @@ async function createAccount(): Promise<void> {
   try {
     step.value = Steps.Creating;
     const start = DateTime.now().toMillis();
-    const access = ParsecAccountAccess.usePassword(creationStepper.email!, choosePasswordRef.value?.password);
+    const access = ParsecAccountAccess.usePassword(creationStepper.email!, choosePasswordRef.value?.password || '');
```

Comment:
No, this is setting the account's authentication, we can't just set an empty string. This error has to be handled properly.